consensus/polybft/extra.go

```diff
@@ -250,9 +256,30 @@ func (d *ValidatorSetDelta) UnmarshalRLPWith(v *fastrlp.Value) error {
 		}
 	}
 
+	// Validators (updated)
+	{
+		validatorsRaw, err := elems[1].GetElems()
+		if err != nil {
+			return fmt.Errorf("array expected for updated validators")
+		}
+
+		if len(validatorsRaw) != 0 {
```

Comment:
This code is the same as for the added validators. Maybe consider extracting some helper function?

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1026/commits/47683e7f9d9dd3a280d08e74e87fc9fc15cc3180